data/about.css

```diff
@@ -98,7 +98,7 @@ button {
 
 .shortcut a img {
     width: 50%;
-    height: 50%;
+    height: 100%;
     margin: auto;
     margin-bottom: .5em;
```

Comment:
You should have changed this one, too.

Reply:
You are right! I will go into detail again ... In the very privacy of my home.